src/tests/additionalSupport.spec.ts

```diff
@@ -0,0 +1,93 @@
+        await loginPage.checkOnPage('HMPPS Digital Services - Sign in')
+        await loginPage.signInWith(UserType.USER_ONE)
+        await homePage.displayBookOrChangeaVisit()
+        await homePage.checkOnPage('Manage prison visits - Manage prison visits')
+        await homePage.selectBookOrChangeVisit()
+    })
+
+    test('Search for a prisoner & book a visit with one visitor and additional support', async ({
+
+        searchPage,
+        prisonerDetailsPage,
+        selectorVisitorPage,
+        selectDateTimePage,
+        additionalSupportPage,
+        mainContactPage,
+        bookingMethodPage,
+        checkYourBookingPage,
+        bookingConfirmationPage
+
+    }) => {
+        test.slow()
+
+        await searchPage.checkOnPage('Manage prison visits - Search for a prisoner')
+        await searchPage.enterPrisonerNumber('A6036DZ')
+        await searchPage.selectPrisonerformResults()
+
+        await prisonerDetailsPage.clickOnBookAPrisonVisit()
+
+        expect(await selectorVisitorPage.checkOnPage('Manage prison visits - Select visitors from the prisoner’s approved visitor list'))
+        await selectorVisitorPage.selectFirstVisitor()
+        await selectorVisitorPage.continueToNextPage()
+
+        expect(await selectDateTimePage.checkOnPage('Manage prison visits - Select date and time of visit'))
+        expect(await selectDateTimePage.headerOnPage('Select date and time of visit'))
+        await selectDateTimePage.selectFirstAvailableSlot()
+        await selectDateTimePage.continueToNextPage()
+
+        expect(await additionalSupportPage.checkOnPage('Manage prison visits - Is additional support needed for any of the visitors?'))
+        expect(await additionalSupportPage.headerOnPage('Is additional support needed for any of the visitors?'))
+        await additionalSupportPage.selectAdditionalSupportRequired()
+        await additionalSupportPage.continueToNextPage()
+
+        await mainContactPage.checkOnPage('Manage prison visits - Who is the main contact for this booking?')
+        expect(await mainContactPage.headerOnPage('Who is the main contact for this booking?'))
+        await mainContactPage.selectMainContactForBooking()
+        await mainContactPage.selectNoPhoneNumberProvided()
+        const mainContact = await mainContactPage.getMainContactName()
+        await mainContactPage.continueToNextPage()
+
+        await bookingMethodPage.checkOnPage('Manage prison visits - How was this booking requested?')
+        expect(await bookingMethodPage.headerOnPage('How was this booking requested?'))
+        await bookingMethodPage.selectBookingMethod()
+        await bookingMethodPage.continueToNextPage()
+
+        await checkYourBookingPage.checkOnPage('Manage prison visits - Check the visit details before booking')
+        expect(await checkYourBookingPage.headerOnPage('Check the visit details before booking'))
+        const mainContactNameOnDetails = await checkYourBookingPage.getMainContactName()
```

Comment:
Because this test is specifically about specifying additional support needs, is it worth checking on `checkYourBookingPage` (and perhaps too on the `bookingConfirmationPage`) that the additional support details have been saved?

Or I guess after booking you could go the booking summary page for the visit and check 🤷‍♂️

Reply:
(I suppose for **any** data we send during the booking journey the way to be most certain that it was processed and saved correctly is to make a subsequent request to the booking summary and see it there.)

Reply:
Addressed the comments , updated the tests as suggested